bigframes/core/io.py

```diff
@@ -16,11 +16,43 @@
 
 import datetime
 import textwrap
-from typing import Dict, Union
+from typing import Dict, Optional, Sequence, Tuple, Union
 
 import google.cloud.bigquery as bigquery
 
 IO_ORDERING_ID = "bqdf_row_nums"
+MAX_LABELS_COUNT = 64
+
+
+def create_job_configs_labels(
+    job_config: Optional[bigquery.job.QueryJobConfig],
```

Comment:
Nit: This might be easier to test if we just pass in the other labels as a dictionary and return the new labels as a dictionary.

Reply:
Pass existing labels as dictionary and return dictionary now.